File: Programing Language/Cpp/rockpaperscissors.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>
using namespace std;

int main() {
    srand(time(0));
    string choices[3] = {"rock", "paper", "scissors"};
    string user;
    int comp = rand() % 3;

    cout << "Enter rock, paper, or scissors: ";
    cin >> user;

    cout << "Computer chose: " << choices[comp] << endl;

    if (user == choices[comp]) {
        cout << "It's a tie!\n";
    }
    else if ((user == "rock" && choices[comp] == "scissors") ||
             (user == "paper" && choices[comp] == "rock") ||
             (user == "scissors" && choices[comp] == "paper")) {
        cout << "You win!\n";
    }
    else {
        cout << "You lose!\n";
    }

    return 0;
}

File: Programing Language/Python/rock_paper_scissors.py
import random

choices = ["rock", "paper", "scissors"]

def play():
    user = input("Choose rock, paper or scissors: ").lower()
    computer = random.choice(choices)
    print(f"Computer chose: {computer}")

    if user == computer:
        print("It's a tie!")
    elif (user == "rock" and computer == "scissors") or \
         (user == "paper" and computer == "rock") or \
         (user == "scissors" and computer == "paper"):
        print("You win!")
    else:
        print("You lose!")

if __name__ == "__main__":
    play()
